server/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.11
```

Comment:
Mind changing the file name to `Containerfile`

Reply:
Done